modules/core/shared/src/main/scala/gem/enum/GsaoiReadMode.scala

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.util.Enumerated
+import java.time.Duration
+
+/**
+ * Enumerated type for GSAOI Read Mode.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GsaoiReadMode(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val ndr: Int,
+  val readNoise: Int,
+  val minimumExposureTime: Duration,
+  val overhead: Int
```

Comment:
This is also a `Duration`.

Reply:
Why is this?

Reply:
I mean why `overhead` is a `Duration`?

Reply:
I believe it is a time in seconds, so we should try to represent it with a more precise type.